### DISCUSSION TITLE
Github rejected my application without giving me a clear reason why.

### DISCUSSION THREAD

**Discussion by SquarishRectangle** [community/community#52864] [GitHub Education]
**Before my current application** I received these errors when I applied incorrectly
![image](https://user-images.githubusercontent.com/61360928/232161857-d761ade4-7f3e-4a13-8dd8-61c31a689402.png)

**Now that I fixed those issues** I am still getting rejected, but instead of giving me clear errors, it links me to [a link containing common issues](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved), none of which apply to me. 
![image](https://user-images.githubusercontent.com/61360928/232162063-a744fc91-3da0-48dd-a991-800171c66070.png)

Because I was instantaneously rejected, it leads me to think there was no manual review, and my application was rejected by the pre-review system. What should I do?

**Comment by samus-aran** [comment]
Hi @SquarishRectangle 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍
